Microsoft/WebTemplateStudio
# Updated the generated package.json for Angular

**PLEASE REVIEW AFTER #1045 IS MERGED**
This is required for running Angular and Node web app on the App Service web app.

To test:
1. Launch the extension in preview mode
2. Create a project using Angular as the frontend framework, Node as the backend
3. Add App Service on the Azure page
4. Create the project
5. Open the project
6. Do Ctrl + Shift + P and type in "Azure App Service: Deploy to Web App..."
7. When deployment is done, click "Browse website"
8. Make sure that the site is working properly
